src/custom/simplifions/components/SimplifionsTags.vue

```diff
@@ -89,6 +68,23 @@ const groupedTags = computed(() => {
 
 const hideBudget = computed(() => props.hideBudget)
 const hideSimplification = computed(() => props.hideSimplification)
+
+const orderedFournisseursDeService = computed(() => {
+  const names = groupedTags.value['fournisseurs-de-service'].map(
+    (fn) => fn.name
+  )
+  const otherItems = names.filter((name) => !name.match(/Autres?/))
+  const autresItems = names.filter((name) => name.match(/Autres?/))
+
+  return [...otherItems.sort(), ...autresItems]
```

Comment:
Don't you want to sort `autresItems` as well, for stability?

Reply:
Not really, I have only one "Autres" actually.